src/DataTypes/Serializations/SerializationNullable.cpp

```diff
@@ -228,7 +228,9 @@ ReturnType SerializationNullable::deserializeTextEscapedImpl(IColumn & column, R
     {
         /// This is not null, surely.
         return safeDeserialize<ReturnType>(column, *nested,
-            [] { return false; },
+            [&istr] {
```

Comment:
This is not correct, the TSV format ("escaped") expect `\N` as NULL, not `NULL`.